saleor/static/dashboard-next/productTypes/views/ProductTypeCreate.tsx

```diff
@@ -60,6 +60,13 @@ export const ProductTypeCreate: React.StatelessComponent = () => {
                     context: "page title"
                   })}
                   saveButtonBarState={loadingCreate ? "loading" : "default"}
```

Comment:
There is a helper function to do that, `getMutationState` from 'saleor/static/dashboard-next/misc.ts`